src/CoreBundle/Interfaces/ChameleonTimeProviderInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Interfaces;
+
+interface ChameleonTimeProviderInterface
+{
+    public function getTime(): int;
+
+    /**
+     * Returns the current DateTime as a DateTime Object. If you provide no time zone, then the systems current timezone
```

Comment:
changed signature - should not be the current date/time anyway since the interface does not know what kind of date/time the implementation will provide.